tox.ini

```diff
@@ -30,6 +30,12 @@ deps =
         -rrequirements/requirements-testing.txt
         -rrequirements/requirements-optionals.txt
 
+[testenv:base]
+; Ensure optional dependencies are not required
+deps =
+        django
+        -rrequirements/requirements-testing.txt
+
```

Comment:
While this looks more sane, my approach allows to use `-base` for everything (i.e. the different Django versions).

Reply:
True, although I'm not sure if it's entirely necessary to have that capability. The basic compat checks are generally going to work, regardless of base Python and Django versions. It should be sufficient just to check once.

In contrast to #6041, testing the optional code was more important, as compatibility between the optional dependencies and the various versions of Python and Django would not be guaranteed.